x/launch/keeper/msg_server_request_add_account.go

```diff
@@ -33,14 +35,33 @@ func (k msgServer) RequestAddAccount(
 		return nil, sdkerrors.Wrap(types.ErrCodecNotPacked, msg.String())
 	}
 
-	requestID := k.AppendRequest(ctx, types.Request{
+	coordAddress, found := k.profileKeeper.GetCoordinatorAddressFromID(ctx, chain.CoordinatorID)
+	if !found {
+		return nil, spnerrors.Critical(
```

Comment:
This one is actually not critical because with `MsgDeleteCoordinator`. The coordinator associated to a chain can be delete

We should still return an error since the request will never beable to be approved since the associated coordinator has been deleted.
We can have a message like 
```
sdkerrors.Wrap(types.ErrChainInactive, "the chain coordinator has been deleted")
```

Reply:
Based on this. We can also add a test for each request when the message fails when the coordinator ID is not associated to an address

Reply:
solves on [ddcd930](https://github.com/tendermint/spn/pull/240/commits/ddcd930d7be0abf48ffb6fb4a49798b59fa299fc)